gradle.properties

```diff
@@ -9,6 +9,8 @@
 
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
+android.enableJetifier=true
+android.useAndroidX=true
```

Comment:
Please move these to an Android-specific section at the bottom of this file (these are not JVM arguments).